ui/raidboss/data/04-sb/eureka/eureka_pyros.js

```diff
@@ -15,17 +15,17 @@
       regexDe: Regexes.startsUsing({ id: '36E0', source: 'Skalli', capture: false }),
       regexFr: Regexes.startsUsing({ id: '36E0', source: 'Sköll', capture: false }),
       regexJa: Regexes.startsUsing({ id: '36E0', source: 'スコル', capture: false }),
-      regexCn: Regexes.startsUsing({ id: '36E0', source: '狼', capture: false }),
+      regexCn: Regexes.startsUsing({ id: '36E0', source: '斯库尔', capture: false }),
```

Comment:
Sorry for that, I added it by copying here. https://github.com/quisquous/cactbot/blob/master/ui/eureka/eureka.js
line 1390, is that wrong? or I didn't pay attention, maybe on eureka tracker is different.
```
label: {
            en: 'Skoll',
            de: 'Skalli',
            fr: 'Sköll',
            ja: 'スコル',
            cn: '狼',
            ko: '스콜',
```

Reply:
you'll see that with @quisquous , I think, but on the https://ffxiv-eureka.com/BvBGkw the name is 狼 for the tracker. 
you have to differentiate the real name in the game for the regex and the name saved in shortcut of the tracker. I know because I had to watch and change everything when I modified everything in French because lots of names were wrong.

Reply:
from the eureka tracker you linked, when i set it to CN, it also shows skolls name as "斯库尔"

Reply:
冷却中的NM: 狼 (119分)
It is not the name of the apparition that you have to look at but the links of the tracker. If you click on copy to clipboard, you will see.